### ISSUE TITLE
get_fpga_buildtime

### ISSUE THREAD

**Issue by alphan** [lowRISC/ot-sca#39]
Please see #36 for context and newaetech/chipwhisperer#306 for what needs to be done.

**Comment by vogelpi** [comment]
Thanks for raising the issue @alphan . I will try to add the build time information to the bitstream.

**Comment by alphan** [comment]
Thanks @vogelpi. Just to clarify, this is not urgent but will definitely help us avoid any surprises and doesn't need an on-disk cache. That's why I thought we should explore this option first.

**Comment by vogelpi** [comment]
Yes, I agree. And it should be fairly easy to enable that. I will have a look and let you know.

**Comment by vogelpi** [comment]
We've discussed this with @vrozic  and @nasahlpa and concluded that this is no longer needed. The timestamp can now be read out but we're no longer really caring about running the latest bitstream but instead specific bitstreams depending on the target IP. In the meantime, we've also added the capability to force a bitstream / skip updating the bitstream from the command line. This is what we really need and also gives us more flexibility.
